### PR TITLE
Comment out GovukContentSecurityPolicy 

### PR DESCRIPTION
**Pull request by kevindew**
Trello: https://trello.com/c/lxxx5XLZ/178-govuk-has-a-half-implemented-content-security-policy-csp

GOV.UK hadn't intended for this app to have the GOV.UK Content Security
Policy yet, with us first planning to roll out this to frontend app. It
looks like this was added as part of an outsourced Rails update [1],
where the dev couldn't have known about our nuanced context.

As this is an app that doesn't receive a lot of developer attention I'm
disabling this as I don't want breaking changes to the CSP [2] to end up
in this app.

[1]: https://github.com/alphagov/travel-advice-publisher/pull/919
[2]: https://github.com/alphagov/govuk_app_config/pull/279

---

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Review by davidgisbey — APPROVED** [review]

File: ` [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Eventually we'll want to use the GOV.UK Content Security Policy in this app,
+# however as of January 2023 we're scoping it to only frontend apps
+#
+# For more info on the GOV.UK CSP see: https://docs.publishing.service.gov.uk/manual/content-security-policy.html
+#
+# GovukContentSecurityPolicy.configure
```
Comment (davidgisbey) [inline `:6]: Is this meant to be here?

Comment (kevindew) [reply `:6]: Do you mean the commented out code? It's intended to remain yes as that's how the CSP is configured.

Comment (davidgisbey) [reply `:6]: As in it's a new file called ```

Comment (kevindew) [reply `:6]: Ha! Oh no 🤦  thank you for spotting that.

Comment (davidgisbey) [reply `]: No worries the rest of it looks good 👍

Comment (kevindew) [reply `]: Super, it's gone now. No idea how I even achieved that. Very pleased you caught it 😅

**Pull request merged**